Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/LMS.yaml

```
@@ -55,6 +70,10 @@ LmsModule:
     findById:
       kvFunction: findOneWithKV
       where: id
+    getAllModulesWithModuleSection:
+      kvFunction: findAllWithOptionsKV
+      where:
+       and: [merchantOperatingCityId, moduleSection]
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 7

(indentation)